source/material_model/steinberger.cc

```diff
@@ -483,7 +512,12 @@ namespace aspect
           max_eta              = prm.get_double ("Maximum viscosity");
           max_lateral_eta_variation    = prm.get_double ("Maximum lateral viscosity variation");
           thermal_conductivity_value = prm.get_double ("Thermal conductivity");
-
+          viscosity_averaging = MaterialUtilities::parse_compositional_averaging_operation ("Viscosity averaging scheme",
+                                prm);
+          // viscosities = Utilities::parse_map_to_double_array (prm.get("Viscosities"),
```

Comment:
Remove commented lines?

Reply:
Thanks a lot John for reviewing my pull request. I will follow what you have suggested. Please let me know for further discussion.